windows/src/desktop/kmshell/web/Keyman.Configuration.System.HttpServer.App.ConfigMain.pas

```diff
@@ -149,13 +155,25 @@ function TConfigMainSharedData.GetHTML: string;
 end;
 
 procedure TConfigMainSharedData.Init(const ABitmapPath, AHTML: string; const AFiles: TStringDynArray);
+var
+  FFile: string;
+  m: TMemoryStream;
 begin
   // Note: we have a separate Init call because we need to have the object to add to the shared data
   // to inject the shared data tag into the HTML. This is a side-effect of rendering the HTML in the
   // form which will be eliminated in a future refactor
-  FBitmapPath := ABitmapPath;
   FHTML := AHTML;
-  FFiles := AFiles;
+  for FFile in AFiles do
+  begin
+    m := TMemoryStream.Create;
+    try
+      m.LoadFromFile(ABitmapPath + FFile);
```

Comment:
A future version of Keyman should remove the need for writing these icons to file altogether. But this is more work than I can justify for beta.